VAMobile/src/components/PhotoUpload.tsx

```diff
@@ -0,0 +1,166 @@
+import { Image, Pressable, PressableProps } from 'react-native'
+import { ImagePickerResponse, launchCamera, launchImageLibrary } from 'react-native-image-picker'
+import { useTranslation } from 'react-i18next'
+import React, { FC, useEffect, useState } from 'react'
+import styled from 'styled-components'
+
+import { NAMESPACE } from 'constants/namespaces'
+import { VAIcon } from './index'
+import { themeFn } from 'utils/theme'
+import { useDestructiveActionSheet, useShowActionSheet } from 'utils/hooks'
+import AsyncStorage from '@react-native-async-storage/async-storage'
+import Box, { BoxProps } from './Box'
+import TextView from './TextView'
+import theme from 'styles/themes/standardTheme'
+
+type PhotoUploadProps = {
+  /** width of the photo */
+  width: number
+  /** height of the photo */
+  height: number
+}
+
+type StyledImageProps = {
+  /** prop to set image width */
+  width: number
+  /** prop to set image height */
+  height: number
+  /** Hardcoded radius of 50 due to circular design plan */
+  borderRadius: number
+  /** Hardcoded radius of 2 due to design plan */
+  borderWidth: number
+  /** Hardcoded border color of white */
+  borderColor: string
+}
+
+const StyledImage = styled(Image)<StyledImageProps>`
+  width: ${themeFn<StyledImageProps>((_theme, props) => props.width)}px;
+  height: ${themeFn<StyledImageProps>((_theme, props) => props.height)}px;
+  border-radius: ${themeFn<StyledImageProps>((_theme, props) => props.borderRadius)}px;
+  border-width: ${themeFn<StyledImageProps>((_theme, props) => props.borderWidth)}px;
+  border-color: ${themeFn<StyledImageProps>((_theme, props) => props.borderColor)};
+`
+//TODO: Add this back to the VeteranStatusScreen when decided upon
+const PhotoUpload: FC<PhotoUploadProps> = ({ width, height }) => {
```

Comment:
I tend to think of "upload" as transmitting a file to a server. Maybe this component could have a name like `SavePhotoToAsyncStorage` to differentiate it from the other places in the app where we do photo upload using an API call.

Reply:
Totally worthwhile. We aren't currently using this component and it may end up just being deleted once we determine a V1/V2 of this feature. I'll leave as is for now with some todos.